examples/hybrid/driver.jl

```diff
@@ -187,14 +197,15 @@ if haskey(ENV, "CI_PERF_SKIP_RUN") # for performance analysis
 end
 
 @info "Running `$test_dir/$test_file_name` test case"
+@info "on a vertical $z_stretch_string grid"
```

Comment:
Suggest including dz_min here in the print

Reply:
Thanks. This does not uses Generalized Exponential Stretching with dz_bottom and dz_top as parameters. Tapio suggested that for these examples we use just the simple EponentialStretching with a fixed height scale. But sure we could add that info. Perhaps in a separate PR